painting/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,19 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:dist="http://schemas.android.com/apk/distribution"
+    package="com.ataulm.artcollector.painting">
+
+    <dist:module
+        dist:onDemand="false"
+        dist:title="@string/title_painting">
+        <dist:fusing dist:include="true" />
+    </dist:module>
+
+    <application>
+        <activity android:name=".ui.PaintingActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
```

Comment:
we'll register the intent filter to handle the correct URI later when we do app links